ember/ember.d.ts

```diff
@@ -1765,9 +1788,43 @@ declare module Ember {
         toArray(): any[];
     }
     module RSVP {
+        interface IPromiseFunction {
+            (value: any):void;
+        }
+
+        interface IPromiseResolverFunction {
+            (resolve: IPromiseFunction, reject: IPromiseFunction): void;

```

Comment:
I think the reject callback should have a slightly different signature, calling out a `reason` instead of a `value`:
```ts
(reason?: any) => void
```
See [Ember's RSVP.Promise docs](http://emberjs.com/api/classes/RSVP.Promise.html).